tez-dag/src/test/java/org/apache/tez/dag/app/rm/TestTaskSchedulerManager.java

```diff
@@ -975,14 +1003,15 @@ TaskScheduler createYarnTaskScheduler(TaskSchedulerContext taskSchedulerContext,
       taskSchedulerContexts.add(taskSchedulerContext);
       testTaskSchedulers.add(yarnTaskScheduler);
       yarnSchedulerCreated.set(true);
+      when(yarnTaskScheduler.getHeldContainersCount()).thenReturn(YARN_TASK_SCHEDULER_HELD_CONTAINERS);
       return yarnTaskScheduler;
     }
 
     @Override
     TaskScheduler createUberTaskScheduler(TaskSchedulerContext taskSchedulerContext, int schedulerId) {
       taskSchedulerContexts.add(taskSchedulerContext);
+      testTaskSchedulers.add(uberTaskScheduler);
```

Comment:
this doesn't have to do anything with this patch, it's just fixed for clarity's sake